spec/view/post_index_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'rails_helper'
+
+RSpec.describe 'Render post index page', type: :system do
+  before do
+    @user = User.create(id: 30, name: 'Tom', photo: 'http://placeimg.com/640/480/any', bio: 'Teacher from Mexico.',
+                        posts_counter: 0)
+    @first_post = Post.create(author_id: @user, title: 'Hello', text: 'This is my first post.', likes_counter: 0,
+                              comments_counter: 0)
+    @second_post = Post.create(author_id: @user, title: 'Hello', text: 'This is my second post.', likes_counter: 0,
+                               comments_counter: 0)
+    Comment.create(text: 'Good boy!', author_id_id: @user, post_id: @first_post.id)
+    Comment.create(text: 'Good boy!', author_id_id: @user, post_id: @second_post.id)
+  end
+
+  describe 'index post page' do
+    it 'displays post text' do
+      visit('/users/30/posts/')
+      expect(page).to have_content('This is my first post.')
+    end
+
+    it 'display the post title' do
+      visit('/users/30/posts/')
+      expect(page).to have_content(@first_post.title)
+    end
+
+    it 'display the post body' do
+      visit('/users/30/posts/')
+      expect(page).to have_content(@first_post.text)
+    end
+
+    it 'display the first comment on a post' do
+      visit('/users/30/posts/')
+      expect(page).to have_content('Comments List')
+    end
+
+    it 'display the how many comments' do
+      visit('/users/30/posts/')
+      expect(page).to have_content(@first_post.comments_counter)
+    end
+
+    it 'display how many Likes' do
+      visit('/users/30/posts/')
+      expect(page).to have_content('Likes: 0')
+    end
```

Comment:
- [ ] Good job so far, but you should also test the next feature:
  - Tests that assert you can see the user's profile picture.
  - Tests that assert you can see the user's username.
  - Tests that assert you can see the number of posts the user has written.
  - Tests that assert you can see a section for pagination if there are more posts than fit on the view.
  - Tests that redirect to the post's show page when you click on a post.

 - [ ] Please kindly add that as well 🙃 🙃